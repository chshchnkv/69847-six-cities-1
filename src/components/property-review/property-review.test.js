import React from "react";
import renderer from "react-test-renderer";
import PropertyReview from "./property-review";

const reviewMock = {
  id: 0,
  user: {
    id: 0,
    is_pro: true,
    name: `Test User`,
    avatar_url: `/images/avatar.jpg`
  },
  rating: 4.3,
  comment: `Liked this place`,
  date: `2019-05-29T08:10:37.218Z`
};

it(`Property review renders correclty`, () => {
  const tree = renderer.create(<PropertyReview id={reviewMock.id} rating={reviewMock.rating} date={reviewMock.date} comment={reviewMock.comment} user={reviewMock.user}/>);
  expect(tree).toMatchSnapshot();
});
