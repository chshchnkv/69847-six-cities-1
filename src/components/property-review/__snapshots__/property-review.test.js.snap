// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property review renders correclty 1`] = `
Array [
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Test User"
        className="reviews__avatar user__avatar"
        height="54"
        src="/images/avatar.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Test User
    </span>
  </div>,
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Liked this place
    </p>
    <time
      className="reviews__time"
      dateTime="2019-05-29T08:10:37.218Z"
    >
      May
       
      29
      , 
      2019
    </time>
  </div>,
]
`;
